pages/index.js

```diff
@@ -261,7 +261,7 @@ class Index extends React.Component {
     console.log(localStorage.getItem("username"))
     if(localStorage.getItem("username")){
       this.setState({
-        loggedIn:true,
+        loggedIn:true/false,
```

Comment:
Ensure that the value assigned to 'loggedIn' is a boolean (true/false) for clarity and consistency.